db/re/item_db_usable.yml

```diff
@@ -49602,6 +49602,22 @@ Body:
     Weight: 10
     Script: |
       /* upgradeui 100745; */
+  - Id: 100783
+    AegisName: MajorAutoSpell_Mix
+    Name: Auto Spell Shadow Pick-Up Box    # !todo check english name
```

Comment:
How about using the name from https://github.com/llchrisll/ROenglishRE/tree/master/Renewal/System

Reply:
It took a while to translate them, don't feel like throwing that work into the bin in exchange for mass google translations.

Reply:
imo you can remove the `# !todo check english name` if you're confident in the name